src/Auth0.OidcClient.Android/SystemBrowserWebView.cs

```diff
@@ -0,0 +1,16 @@
+using Android.App;
+using Android.Content;
+
+namespace Auth0.OidcClient
+{
+    public class SystemBrowserWebView : AndroidWebViewBase
+    {
+        protected override void LaunchBrowser(Android.Net.Uri uri)
+        {
+            var intent = new Intent(Intent.ActionView, uri);
+            intent.AddFlags(ActivityFlags.NoHistory)
```

Comment:
Do you offer a way to disable custom tabs? if not, why are you keeping this? By default if the custom tabs intent fails to find a compatible custom-tabs-browser, it will use any browser app that can resolve an ActionView intent.

Reply:
Wanted to keep it around for a while for backward-compatibility in case people have problems with custom chrome tabs.

Reply:
@damieng Ok. But mark it as deprecated at least so they are aware it's going to be removed soon.

Reply:
Will deprecate it in a couple of months.  (Deprecated attribute is often a compiler error in .NET as people turn on "treat warnings as errors")